1.9.0/values.yaml

```diff
@@ -0,0 +1,159 @@
+# Default values for DirectorOnPrem.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# Update platfrom according to your Platform, Default include(GKE,Packet) and SUSE
+platform: Default
+
+server:
+  release: 1.9.0-ee
+  dockerSecret: directoronprem-registry-secret
+  protocol: http
+  url:
+  httpNodePort: 30380
+  httpsNodePort: 30381
+  apiAuthAccessMode: unrestricted
+  serverDefaultAccessGrant: true
+  apiUiEnabled: true
+  maxMemberCountInOneProject: 10
+  setupName: directoronprem
+  apiAuthExternalProviderEnabled: false
+  apiAuthExternalProviderConfigured:
+  apiAuthInternalProviderEnabled: true
+  apiAuthInternalProviderConfigured: localAuthConfig
+  apiAuthGithubClientId:
+  apiAuthGithubClientSecret:
+  apiAuthGoogleClientId:
+  apiAuthGoogleClientSecret:
+  clusterDomain: cluster.local
+  featureSubscriptionDisable: false
+  subscriptionType: internal
+  featureBillingDisable: true
+  billingType: external
+  featureEmailDisable: true
+  senderEmailAddress:
+  senderEmailPassword:
+  featureDocsDisable: false
+  docsUrl: https://help.mayadata.io/hc/en-us
+  featureChatBotDisable: true
+  slackConfigBotClientId:
+  slackConfigBotClientSecret:
+  slackNotificationWelcomeMessage:
+  featureKialiDisable: true
+  analyticsGoogleCode:
+  ## featureAnalyticsDisable should be set to false in director-charts public repo
+  featureAnalyticsDisable: true
```

Comment:
by default should it be enabled?

Reply:
Good point @shovanmaity .

Reply:
On user request, it was disabled from last release.